client/jetpack-connect/plans-extended-info.jsx

```diff
@@ -0,0 +1,51 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormattedHeader from 'components/formatted-header';
+
+const PlansExtendedInfo = ( { recordTracks, translate } ) => {
+	return (
+		<Card className="jetpack-connect__plan-info">
+			<img
+				className="jetpack-connect__plan-info-illustration"
+				src="/calypso/images/illustrations/jetpack-start.svg"
+				alt=""
+			/>
+			<FormattedHeader
+				headerText={ translate( 'Unsure which plan is right for you?' ) }
+				subHeaderText={ translate(
+					'Read our guide to choosing the right plan or compare plans in more detail.'
+				) }
+			/>
+			<div className="jetpack-connect__plan-info-buttons">
+				<Button
+					primary
+					href="https://jetpack.com/2017/02/01/a-simple-guide-to-choosing-the-best-jetpack-plan-your-wordpress-site/"
```

Comment:
Looks a little weird to be linking to a post here. Should we turn this to a page on jetpack.com?

Reply:
Designs :)

Reply:
Let's ping someone to take care of this. Maybe @nicoleckohler?

Reply:
I thought the same and had asked Growth about this and they recommended those posts—they are the same links as on https://jetpack.com/pricing/ below the plans grid.